plugin/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "gatsby-source-shopify-experimental",
-  "version": "1.14.1",
+  "name": "@gatsbyjs/gatsby-source-shopify",
+  "version": "2.0.0-rc.x",
```

Comment:
When using automation it can actually get tricky to keep the package.json in sync with the published npm version. 

There are some GitHub actions that will allow you to commit files that are changed during the workflow. So one option is to use an action like that and commit the package.json after the workflow runs `npm version <version_number>`.

But to keep things simple, I am choosing to leave this as `.x` for now. So publishing will work like this:

1. Check the last prerelease version in npm. For this example, say it's 2.0.0-rc.0
2. [Create a new release in GitHub](https://github.com/gatsbyjs/gatsby-source-shopify-experimental/releases/new) with the version number v2.0.0-rc.1
3. Now the action will automatically publish v2.0.0-rc.1

If you format the tag incorrectly, this action will not run. And if you try to publish a previously released version, the action will fail to publish. So this should be good enough to keep rolling the release candidate forward.